pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerializeJavaR.scala

```diff
@@ -0,0 +1,297 @@
+package edu.berkeley.cs.amplab.sparkr
+
+import scala.collection.JavaConversions._
+
+import java.io.DataInputStream
+import java.io.DataOutputStream
+
+/**
+ * Utility functions to serialize, deserialize objects to / from R
+ */
+object SerializeJavaR {
```

Comment:
Hmm okay - I'll make it SerDe for now (the R and JVM part are kind of obvious I guess being in SparkR).